packages/datagateway-download/src/downloadTab/downloadTab.component.tsx

```diff
@@ -94,23 +96,39 @@ const DownloadTabs: React.FC = () => {
         textColor="primary"
         centered
       >
-        <Tab label="Cart" aria-label="Cart tab" {...a11yProps(0)} />
-        <Tab label="Downloads" aria-label="Downloads tab" {...a11yProps(1)} />
+        <Tab
+          label="Cart"
```

Comment:
Should the tab labels be also 'translated' as well as the aria-labels?

Reply:
Totally missed this, yes they should be!